packages/libs/react-ui/CONTRIBUTING.md

```diff
@@ -52,6 +52,8 @@ more subcomponents would negatively affect DX.
 > [here](https://bradfrost.com/blog/post/atomic-web-design/) and
 > [here](https://atomicdesign.bradfrost.com/chapter-2/).
 
+#### Button component as an example
```

Comment:
This is the same heading level as `Composition - Subcomponent structure` which is a section I think it should be a child of

Reply:
Ah yes! Will change